workflows/sample/tasks/hifiasm.wdl

```diff
@@ -66,17 +66,18 @@ task hifiasm_assemble {
     (hifiasm -o ~{prefix} -t ~{threads} ~{sep=" " movie_fasta}) > ~{log_name} 2>&1
   >>>
   output {
-    File a_ctg        = "~{prefix}.a_ctg.gfa"
-    File a_ctg_noseq  = "~{prefix}.a_ctg.noseq.gfa"
-    File p_ctg        = "~{prefix}.p_ctg.gfa"
-    File p_ctg_noseq  = "~{prefix}.p_ctg.noseq.gfa"
-    File p_utg        = "~{prefix}.p_utg.gfa"
-    File p_utg_noseq  = "~{prefix}.p_utg.noseq.gfa"
-    File r_utg        = "~{prefix}.r_utg.gfa"
-    File r_utg_noseq  = "~{prefix}.r_utg.noseq.gfa"
-    File ec_bin       = "~{prefix}.ec.bin"
-    File ovlp_rev_bin = "~{prefix}.ovlp.reverse.bin"
-    File ovlp_src_bin = "~{prefix}.ovlp.source.bin"
+    File hap1_p_ctg        = "~{prefix}.bp.hap1.p_ctg.gfa"
+    File hap1_p_ctg_lowQ   = "~{prefix}.bp.hap1.p_ctg.lowQ.bed"
+    File hap1_p_noseq      = "~{prefix}.bp.hap1.p_ctg.noseq.gfa"
+    File hap2_p_ctg        = "~{prefix}.bp.hap2.p_ctg.gfa"
+    File hap2_p_ctg_lowQ   = "~{prefix}.bp.hap2.p_ctg.lowQ.bed"
+    File hap2_p_noseq      = "~{prefix}.bp.hap2.p_ctg.noseq.gfa"
+    File p_ctg             = "~{prefix}.bp.p_ctg.gfa"
+    File p_utg             = "~{prefix}.bp.p_utg.gfa"
+    File r_utg             = "~{prefix}.bp.r_utg.gfa"
+    File ec_bin            = "~{prefix}.ec.bin"
+    File ovlp_rev_bin      = "~{prefix}.ovlp.reverse.bin"
+    File ovlp_src_bin      = "~{prefix}.ovlp.source.bin"
```

Comment:
There's no reason to keep these outputs (*.p_utg.gfa, *.r_utg.gfa, `*.bin`).  We also might not need to keep `~{prefix}.bp.p_ctg.gfa`.  We should check with CMKC on that.

Also, I'm not sure how straightforward this is, but we don't need to keep these after running gfa2fasta:
`{prefix}.bp.hap1.p_ctg.gfa`
`{prefix}.bp.hap2.p_ctg.gfa`

Reply:
So because we've designed this for cloud native implementation- by default we save the pointers to all intermediate files in the output.json (that's what's actually why we put anything in the "output" section of the WDL)- it becomes very difficult to find those files otherwise.

CMH has data retention policies in place that will automatically purge intermediate file results and will leverage CROO (to copy specific files they want to destination with specified naming schema). any intermediate files will not be gathered and purged automatically.